src/Entity/Activity/Activity.php

```diff
@@ -38,6 +38,11 @@ class Activity
     #[GQL\Description('The name of the activity.')]
     private $name;
 
+    #[ORM\Column(name: 'isArchived', type: 'boolean', options: ['default' => false])]
+    #[GQL\Field(type: 'Boolean!')]
+    #[GQL\Description('If this activity is archived')]
+    private bool $isArchived;
```

Comment:
It is common practice to let the property be named `archived` and its accessors be named `isArchived` and `setArchived` instead (see for example the [Symfony PropertyAccess component](https://symfony.com/doc/current/components/property_access.html#using-hassers-issers)). Please update this class accordingly.

Reply:
didn't know this one, but it makes sense. I'll edit it